services/glance/README.md

```diff
@@ -0,0 +1,14 @@
+# Glance with Tailscale Sidecar Configuration
+
+This Docker Compose configuration sets up [Glance](https://github.com/glanceapp/glance) with Tailscale as a sidecar container to securely access your system monitoring dashboard over a private Tailscale network. By using Tailscale in a sidecar configuration, you can enhance the security and privacy of your Glance instance, ensuring that it is only accessible within your Tailscale network.
+
+## Glance
+
+[Glance](https://github.com/glanceapp/glance) is a sleek, real-time dashboard for monitoring your system metrics, Docker containers, and other self-hosted services. It offers a clean and responsive interface that consolidates key system stats and service statuses in one place. This configuration uses Tailscale to securely expose your Glance instance, keeping it protected from the public internet and accessible only within your private Tailscale network.
+
+To install Glance properly, please follow the official instructions at:
+👉 [Glance Installation Guide](https://github.com/glanceapp/glance?tab=readme-ov-file#installation)
```

Comment:
Why do we refer to external documentation? This repo should contain a ready-to-go deployment yaml only.